sdk/identity/azure-identity/src/main/java/com/azure/identity/IntelliJCredentialBuilder.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity;
+
+import com.azure.core.util.CoreUtils;
+
+import java.util.Objects;
+
+/**
+ * Fluent credential builder for instantiating a {@link IntelliJCredential}.
+ *
+ * @see IntelliJCredential
+ */
+public class IntelliJCredentialBuilder extends CredentialBuilderBase<IntelliJCredentialBuilder> {
```

Comment:
This should be removed when we hide the credential class. Any configuration from this builder will have to be added to the `DefaultAzureCredentialBuilder`.

Reply:
Why this design though? Not being able to construct individual credentials has been considered a bug by our customers: https://github.com/Azure/azure-sdk-for-java/issues/9162